package.go

```diff
@@ -0,0 +1,3 @@
+// Gostats is a Golang statistics library created by Engineers at Lyft
```

Comment:
package level comments should start with a short label and match the actual package name (not the repo/directory name): `// The stat package provides ...`